### DISCUSSION TITLE
Different item in field.json without "xsdSequenceOrder" attribute SDK 1.7

### DISCUSSION THREAD

**Discussion by jaggaer-fgeniola** [OP-TED/eForms-SDK#543] [Q&A] [answered]
Hi,
I'm using the information "xsdSequenceOrder" to correctly order the elements during the creation of the xml but for some items (eg: efac:Changes) it is not reported so it is not possible for me to build the xml correctly.

Is there an alternative way to know the exact order of the xml elements to create?

Thank you

**Comment by bertrand-lorentz** [comment, marked as answer]
Thanks for your feedback.
The missing "xsdSequenceOrder" properties are a known issue in SDK 1.7.0 ( see issue #490 ), and it will be corrected in the upcoming SDK 1.8.0.
